schemas/log.yml

```diff
@@ -2,8 +2,14 @@
 - name: log
   title: Log
   group: 2
+  short: Low level details specific to log events.
```

Comment:
Suggest:  "Details about the event's logging mechanism."   <I prefer "logging mechanism" vs. "log events" since metrics can be carried over syslog as well, as mentioned in  https://github.com/elastic/ecs/pull/301#issuecomment-455617131)>